api/src/utils/aggregate-utils.ts

```diff
@@ -51,6 +52,19 @@ const aggregatePitch = async (rawPitch: IPitch): Promise<IPitchAggregate> => {
     })),
   );
 
+  const interests = await Promise.all(
+    rawPitch.topics.map(async (topic) => {
+      const interest = await Interest.findById(topic);
+
+      return {
+        _id: interest._id,
+        name: interest.name,
+        color: interest.color,
+        active: interest.active,
+      };
+    }),
+  );
+
```

Comment:
I feel like we should replace this with the `.then` like is done for `teams` below. It should be faster in theory.